index.html

```diff
@@ -1,21 +1,152 @@
+          />
+          <div class="main-name">
+            Maya <br />
+            Nelson
+          </div>
+          <hr class="blue-line" />
+          <div class="main-title">PROJECT MANAGER</div>
+          <div class="contact">
+            <img
+              alt="Facebook"
+              src="https://static.wixstatic.com/media/0fdef751204647a3bbd7eaa2827ed4f9.png/v1/fill/w_25,h_25,al_c,q_85,usm_0.66_1.00_0.01,enc_auto/0fdef751204647a3bbd7eaa2827ed4f9.png"
+              style="width: 25px; height: 25px; object-fit: cover"
+            />
+            <img
+              alt="Twitter"
+              src="https://static.wixstatic.com/media/c7d035ba85f6486680c2facedecdcf4d.png/v1/fill/w_25,h_25,al_c,q_85,usm_0.66_1.00_0.01,enc_auto/c7d035ba85f6486680c2facedecdcf4d.png"
+              style="width: 25px; height: 25px; object-fit: cover"
+            />
+            <img
+              alt="LinkedIn"
+              src="https://static.wixstatic.com/media/6ea5b4a88f0b4f91945b40499aa0af00.png/v1/fill/w_25,h_25,al_c,q_85,usm_0.66_1.00_0.01,enc_auto/6ea5b4a88f0b4f91945b40499aa0af00.png"
+              style="width: 25px; height: 25px; object-fit: cover"
+            />
+            <img
+              alt="Instagram"
+              src="https://static.wixstatic.com/media/01c3aff52f2a4dffa526d7a9843d46ea.png/v1/fill/w_25,h_25,al_c,q_85,usm_0.66_1.00_0.01,enc_auto/01c3aff52f2a4dffa526d7a9843d46ea.png"
+              style="width: 25px; height: 25px; object-fit: cover"
+            />
+          </div>
+        </div>
+        <div class="hello-part">
+          <div class="hello-text">Hello</div>
+          <div class="role">Here's who I am & what I do</div>
+          <div class="btn">
+            <button class="resume-btn">RESUME</button>
+            <button class="projects-btn">PROJECTS</button>
+          </div>
+          <p class="p1">
+            I'm a paragraph. Click here to add your own text and edit me. It’s
+            easy. Just click “Edit Text” or double click me to add your own
+            content and make changes to the font.
+          </p>
+          <p class="p2">
+            I’m a great place for you to tell a story and let your users know a
+            little more about you.
+          </p>
+        </div>
+      </div>
+    </main>
+    <footer>
+      <p class="purpose1">
+        This website design was created by Wix.com, and is used here for
+        strictly educational purposes.
+      </p>
+      <div class="footer-call">
+        <div class="footer-major">Call</div>
+        <a class="footer-contain" href="tel:+441234567890">123-456-7890</a>
+      </div>
+      <div class="footer-write">
+        <div class="footer-major">Write</div>
+        <a class="footer-contain" href="mailto:info@mysite.com"
+          >info@mysite.com</a
+        >
+      </div>
+      <div class="footer-follow">
+        <div class="footer-major">Follow</div>
+        <div class="footer-contact">
+          <img
```

Comment:
What might these images be meant to do? Is there another element that might offer more functions?